package/MDAnalysis/analysis/align.py

```diff
@@ -380,6 +377,175 @@ def alignto(mobile, reference, select="all", mass_weighted=False,
+             described under :ref:`ordered-selections-label`).
+          *filename*
+             file name for the RMS-fitted trajectory or pdb; defaults to the
+             original trajectory filename (from *traj*) with *prefix* prepended
+          *rmsdfile*
+             file name for writing the RMSD timeseries [``None``]
+          *prefix*
+             prefix for autogenerating the new output filename
+          *mass_weighted*
+             do a mass-weighted RMSD fit
+          *tol_mass*
+             Reject match if the atomic masses for matched atoms differ by more than
+             *tol_mass* [0.1]
+
+          *strict*
+             Default: ``False``
+             - ``True``: Will raise :exc:`SelectioError` if a single atom does not
+               match between the two selections.
+             - ``False``: Will try to prepare a matching selection by dropping
+               residues with non-matching atoms. See :func:`get_matching_atoms`
+               for details.
+          *force*
+             - ``True``: Overwrite an existing output trajectory (default)
+             - ``False``: simply return if the file already exists
+          *quiet*
+             - ``True``: suppress progress and logging for levels INFO and below.
+             - ``False``: show all status messages and do not change the the logging
+               level (default)
+
+
+
+
+          *kwargs*
+             All other keyword arguments are passed on the trajectory
+             :class:`~MDAnalysis.coordinates.base.Writer`; this allows manipulating/fixing
+             trajectories on the fly (e.g. change the output format by changing the extension of *filename*
+             and setting different parameters as described for the corresponding writer).
+
+
+            .. _ClustalW: http://www.clustal.org/
+            .. _STAMP: http://www.compbio.dundee.ac.uk/manuals/stamp.4.2/
+
+            .. versionchanged:: 0.8
+               Added *kwargs* to be passed to the trajectory :class:`~MDAnalysis.coordinates.base.Writer` and
+               *filename* is returned.
+
+            .. versionchanged:: 0.10.0
+               Uses :func:`get_matching_atoms` to work with incomplete selections
+               and new *strict* keyword. The new default is to be lenient whereas
+               the old behavior was the equivalent of *strict* = ``True``."""
+
+        frames = u.trajectory
+        self._setup_frames(frames, start,stop,step)
+        self.rmsdfile = rmsdfile
+
+        if quiet:
+            # should be part of a try ... finally to guarantee restoring the log level
+            logging.disable(logging.WARN)
+
+        kwargs.setdefault('remarks', 'RMS fitted trajectory to reference')
+
+        if filename is None:
+            path, fn = os.path.split(frames.filename)
+            self.filename = os.path.join(path, prefix + fn)
+            _Writer = frames.Writer
```

Comment:
Is it not simpler to use `mda.Writer(self.filename)`? (which iirc goes through `get_writer_for`) or does `Reader.Writer` do something different?